### PR TITLE
Added new weapon to act as intermediate step between crude sword and real swords like the broadsword

### PR DESCRIPTION
**Pull request by Malorn-Deslor**
#### Summary
Content "Added new weapon to act as intermediate step between crude sword and real swords like the broadsword"

#### Purpose of change

While playing a sword based character, I noticed that after you make a crude sword, which can be made at fabrication 2 with only a hammer and very basic materials... you are done until you find 'The historic weaponsmith' and can make actual fully functional amazing swords. Or until you find one, of course. 

I felt that was a little unreasonable, since the crude sword is just scrap metal hammered over wood, and thus is incredibly crude and simple. It seemed reasonable for a character who now had high fabrication and fully blacksmithing abilities to be able to make something a bit better, even if they hadn't found any manuals on swordsmithing.

#### Describe the solution

I created a new item called 'metal sword' which is still very inferior to any of the historic weapons, even the broadsword. It is basically just the attempt of a skilled (fabrication 7) blacksmith to create a sword, just based on what they know swords are supposed to look like.

It does 5 more cutting damage then a crude sword (23 cutting), is better at blocking, and is made of steel not wood--otherwise it does not differ greatly. Broadswords are the weakest historic sword, and they do 27 cutting damage, so this felt like a nice mid-point. It blocks as well as a broadsword, but it's to-hit is lacking and equal to the crude sword. The metal sword requires high fabrication, and all the normal metalworking tools. It uses materials similar to a broadsword crafting requirements.

Of course, it is usable with medieval swordsmanship, that was the entire point of the item.

#### Describe alternatives you've considered

Well... I considered not having any weapon upgrade from the time you find a hammer until the time you have full metalworking and a very rare book... but I didn't like that option. There are many more powerful weapons that can be made without any book recipes at fabrication 7 with full metalworking tools. So it's nice to give a mid-range upgrade to the medieval swordsmanship path.

#### Testing

I loaded the game up, checked to see if the item was craftable, crafted it, wielded it, attacked a zombie with it. Worked as expected.

### CODE REVIEW

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -53,6 +53,25 @@
     "qualities": [ [ "BUTCHER", -50 ] ],
     "techniques": [ "WBLOCK_1" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "sword_metal",
+    "symbol": "!",
+    "color": "light_gray",
+    "name": { "str": "metal sword" },
+    "description": "An imitation of a true sword forged by someone who knows what swords look like, at least. It has the general shape of a sword, and it is sharp, yet the quality is lacking and the balance isn't quite right.",
```
Comment (BrettDong) [inline data/json/items/melee/swords_and_blades.json:62]:
```suggestion
    "description": "An imitation of a true sword forged by someone who knows what swords look like, at least.  It has the general shape of a sword, and it is sharp, yet the quality is lacking and the balance isn't quite right.",
```

**Comment by Termineitor244** [comment]
Please add this sword to Ninjutsu and Niten Ichi-Ryu, both styles more than qualify to use this weapon.

I'm working on the weapon categories for Ninjutsu, so all of these would be easier in the future.

**Comment by Malorn-Deslor** [comment]
I'll check for the crude sword in either of them, but if the crude sword works for this, I see no reason to not do that...

**Comment by Termineitor244** [comment]
Thanks! And it has more to do with if the style would work with the weapon in question or not, but anyways, those long lists of weapons in the martial arts json are things of the past! With weapon categories, the only thing that will matter is if the martial art can use the kind of weapon in question.

**Comment by Wokko1** [comment]
Please add looks_like for tilesets

**Comment by Malorn-Deslor** [comment]
> "looks_like": "sword_crude"

That's a wonderful idea, thank you.

**Comment by Malorn-Deslor** [comment]
Hmm... it looks like the checks are not running themselves properly, I'm going to try to close and reopen the PR.

**Comment by Malorn-Deslor** [comment]
Does anyone know how to make it possible to dissemble an item without adding an uncrafting recipe? Many other weapons can be broken down into scrap despite not having an explicit recipe? I'd like the new sword to be something that can be broken down.

**Comment by anothersimulacrum** [comment]
If the crafting recipe is reversible, it can be used as both a crafting and uncrafting recipe.

File: data/json/items/melee/swords_and_blades.json [hunk 2]
```diff
@@ -53,6 +53,26 @@
...
+    "looks_like": "sword_crude",
+    "material": [ "steel" ],
+    "volume": "2 L",
+    "weight": "1100 g",
+    "longest_side": "100 cm",
+    "bashing": 6,
+    "cutting": 23,
+    "to_hit": 1,
+    "price_postapoc": 500,
+    "flags": [ "SHEATH_SWORD" ],
```
Comment (Termineitor244) [inline data/json/items/melee/swords_and_blades.json:72]:
```suggestion
    "flags": [ "SHEATH_SWORD" ],
    "weapon_category": [ "MEDIUM_SWORDS" ],
```

File: data/json/martialarts.json [hunk 3]
```diff
@@ -903,6 +904,7 @@
       "switchblade",
       "sword_bayonet",
       "sword_crude",
+      "sword_metal",
```
Comment (Termineitor244) [inline data/json/martialarts.json:907]:
```suggestion
```

**Review by Termineitor244 — CHANGES_REQUESTED** [review]
Sorry, the weapon categories for Ninjutsu were merged first.

**Comment by Malorn-Deslor** [comment]
> Sorry, the weapon categories for Ninjutsu were merged first.

No worries!

**Comment by kevingranade** [comment]
No issue with the content of the PR, but it looks like Github Actions has been disabled for your repository, see https://github.com/Malorn-Deslor/Cataclysm-DDA/actions

It's possible you fiddled with switching your repository to private or something at some point, which will enable billiong, and if you don't pay it'll disable features of your repository.  That's PURE speculation though.
Options you can pursue:
1. Contact Github support by following the link on the above page.
2. Just make a new repository and who knows, an account? I believe a similar issue happened here: https://github.com/CleverRaven/Cataclysm-DDA/pull/51389

**Comment by Malorn-Deslor** [comment]
> 
> 
> No issue with the content of the PR, but it looks like Github Actions has been disabled for your repository, see https://github.com/Malorn-Deslor/Cataclysm-DDA/actions
> 
> It's possible you fiddled with switching your repository to private or something at some point, which will enable billing, and if you don't pay it'll disable features of your repository. That's PURE speculation though.
> Options you can pursue:
> 
>     1. Contact Github support by following the link on the above page.
> 
>     2. Just make a new repository and who knows, an account? I believe a similar issue happened here: [Mythos Mod - Start of conversion, remove zombies from zombie game! #51389](https://github.com/CleverRaven/Cataclysm-DDA/pull/51389)

I didn't much with making it private or anything like that. The good news is that it is telling that actions are disabled and I should contact GitHub support. So I did that. Thank you for the warning and the information.

Is that the reason that the automatic checks for this PR are 'Waiting for status to be reported'?

**Comment by kevingranade** [comment]
Yes, for whatever reason it seems that if there's an issue with your account it impacts pull requests you open on other repositories as well, and it's causing our checks to not run.

**Comment by Malorn-Deslor** [comment]
I am attempting to talk to github support at this point. It's not really working out very well, since they just linked the entire terms and conditions of use, and said I violated it in some unspecified fashion. If anyone wants to mirror the pull request, feel free.

I'm sorry this is causing so many problems. It's strange, this account exists only to contribute to this repo, so I'm not really sure what's going on.

**Comment by BrettDong** [comment]
Mirrored in #51970.

**Comment by BrettDong** [comment]
Merged in #51970.

**Pull request closed without merging**